src/main/drivers/pwm_mapping.h

```diff
@@ -36,10 +36,10 @@
 #define MAX_INPUTS  8
 
 #define PWM_TIMER_MHZ 1
-//these three have to be the same because of the ppmAvoidPWMTimerClash functions
-#define ONESHOT_TIMER_MHZ 12
+#define ONESHOT125_TIMER_MHZ 8
+#define ONESHOT42_TIMER_MHZ 24
```

Comment:
The only drawback to running OS42 at 12 MHz is we cut the resolution in half. My testing has actually shown this to give better flight characteristics. The KISS ESCs run better with a 250 steps of resolution than they do at 1000 steps of resolution. They jitter back and forth less.